addons/website/static/src/interactions/carousel/carousel_section_slider.edit.js

```diff
@@ -1,38 +0,0 @@
-import { Interaction } from "@web/public/interaction";
```

Comment:
Keep the file for now, if you see that it doesn't break html_builder to keep it
(but I think we should keep it anyway for now in order for website to still work) :+1: 

Maybe add a big "TODO remove this file when removing website"